master/internal/webhooks/api_webhook.go

```diff
@@ -26,15 +27,22 @@ type WebhooksAPIServer struct{}
 // AuthorizeRequest checks if the user has CanEditWebhooks permissions.
 // TODO remove this eventually since authz replaces this
 // We can't yet since we use it else where.
-func AuthorizeRequest(ctx context.Context) error {
+func authorizeEditRequest(ctx context.Context, workspaceID int32) error {
 	curUser, _, err := grpcutil.GetUser(ctx)
 	if err != nil {
 		return status.Errorf(codes.Internal, "failed to get the user: %s", err)
```

Comment:
This isn't code that you wrote, but should we change this to 
```suggestion
		return status.Errorf(codes.Internal, "failed to get the user: %w", err)
```

Reply:
I believe `status.Errorf` does not work with `%w`, we can change it to `%v` though

Reply:
Awesome!